### PR TITLE
Feature/final milestone

### PR DESCRIPTION
**Pull request by LOctavio**
Add navigation bar
Add footer
Complete stylings

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,36 +9,82 @@
...
+        <li class="nav-item">
+          <a class="nav-link"  onclick="document.getElementById('list-page').style.display = 'block'; document.getElementById('book-form').style.display = 'none'; document.getElementById('contact-page').style.display = 'none';">List <span class="ml-3 text-primary">|</span> </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" onclick="document.getElementById('list-page').style.display = 'none'; document.getElementById('book-form').style.display = 'block'; document.getElementById('contact-page').style.display = 'none';">Add new <span class="ml-3 text-primary">|</span></a>
+        </li>
+        <li class="nav-item" >
+          <a class="nav-link" onclick="document.getElementById('list-page').style.display = 'none'; document.getElementById('book-form').style.display = 'none'; document.getElementById('contact-page').style.display = 'block';">Contact</a>
+        </li>
+      </ul>
```
Comment (voscarmv) [inline index.html]: - [x] Good use of `display` styles here. But it's preferable to use [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) to display/hide content. Please use this instead here.

File: index.html [hunk 2]
```diff
@@ -9,36 +9,82 @@
...
+          <tbody id="book-list" class="mb-3"></tbody>
+        </table>
+    </div>
+    
+    <form id="book-form">
+      <h4 class="text-center mt-3">Add a new Book!</h4>
 
     <div class="align">
-    <form id="book-form">
       <div class="form-group">
```
Comment (voscarmv) [inline index.html:53]: - [ ] **[OPTIONAL]** Let's work on our HTML indentation. This is only one place where the indentation of nested HTML tags is unorganized. You may quickly fix your entire HTML document by formatting it with something like Visual Studio Code.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @LOctavio , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- Only a few changes needed regarding the method we use to show/hide content.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Improve your HTML indentation.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @LOctavio @Gracetemitope🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

doing good

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**